app/timetables/models.py

```diff
@@ -175,3 +175,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Vendor(SlugifyMixin, TimestampMixin):
+    """Model representing food service-provider"""
+
+    name = models.CharField(max_length=255)
```

Comment:
@andela-kerinoso You're very correct. `null` and `blank` options are set to False by default